.env.example

```
@@ -10,3 +10,14 @@ RDS_PORT=
 
 STATIC_ROOT=
 MEDIA_ROOT=
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)